pages/linux/ncdu.md

```diff
@@ -0,0 +1,19 @@
+# ncdu
+
+> Disk usage analyzer with an ncurses interface.
+
+- Analyze the current working directory:
+
+`ncdu`
+
+- Analyze a given directory:
+
+`ncdu {{path/to/directory}}/`
```

Comment:
Could you remove the trailing slash here?

Reply:
done